shell/platform/tizen/flutter_tizen.cc

```diff
@@ -13,26 +13,47 @@
 #include "flutter/shell/platform/tizen/logger.h"
 #include "flutter/shell/platform/tizen/public/flutter_platform_view.h"
 
+namespace {
+
 // Returns the engine corresponding to the given opaque API handle.
-static flutter::FlutterTizenEngine* EngineFromHandle(
```

Comment:
I still didn't understand why this has the same as before.(Even though I read the article you gave me)

Reply:
Article? If I understand you correctly, this is just a coding style difference. In short, `static` is C style and the anonymous namespace is C++ style. I prefer using an anonymous namespace if there are more than one or two static variables/functions in a file.